geode-core/src/main/java/org/apache/geode/distributed/internal/membership/InternalDistributedMember.java

```diff
@@ -858,7 +858,7 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
     String name = DataSerializer.readString(in);
     this.uniqueTag = DataSerializer.readString(in);
     String durableId = DataSerializer.readString(in);
-    int durableTimeout = DataSerializer.readInteger(in).intValue();
+    int durableTimeout = in.readInt();
```

Comment:
Instead of using DataInput.readInt, it is probably better to use DataSerializer.readPrimitiveInt.

Reply:
I see that this method has a bunch of other places in it that use DataInput.readInt so I will stick with that change.